saleor/graphql/account/tests/mutations/test_token_verify.py

```diff
@@ -0,0 +1,53 @@
+from .....account.error_codes import AccountErrorCode
+from .....core.jwt import create_access_token
+from ....tests.utils import get_graphql_content
+
+MUTATION_TOKEN_VERIFY = """
+    mutation tokenVerify($token: String!){
+        tokenVerify(token: $token){
+            isValid
+            user{
+              email
+            }
+            accountErrors{
+              code
+            }
+        }
+    }
+"""
+
+
+def test_verify_token(api_client, customer_user):
+    variables = {"token": create_access_token(customer_user)}
+    response = api_client.post_graphql(MUTATION_TOKEN_VERIFY, variables)
+    content = get_graphql_content(response)
+    data = content["data"]["tokenVerify"]
+    assert data["isValid"]
```

Comment:
```suggestion
    assert data["isValid"] is True
```

Reply:
Why? `assert data["isValid"]` seems clear and if I'm correct also pylint or some other linter recommends that. Same with `assert not data["isValid"]`.

Reply:
`assert not data["isValid"]` 
This test will pass if `isValid` is `False` or `None` but we need `False` in this case.

Reply:
I guess both `None` and `False` would mean the same here.